uefi/flashregionsection.go

```diff
@@ -54,10 +68,10 @@ func (f FlashRegionSection) Summary() string {
 		"    PdrBase=%v (size: %v)\n"+
 		"}",
 		strings.Join(f.AvailableRegions(), ","),
-		f.BiosBase, f.BiosLimit,
-		f.MeBase, f.MeLimit,
-		f.GbeBase, f.GbeLimit,
-		f.PdrBase, f.PdrLimit,
+		f.BIOS.Base, f.BIOS.Limit,
```

Comment:
Since Region has a fmt.Stringer, maybe just fmt.Sprintf(..."BIOS=%v"..., ...f.BIOS...)?

Reply:
Lets do this another pr